.github/workflows/operator-integration-test.yml

```diff
@@ -68,24 +68,43 @@ jobs:
             echo "Pod is created test case: PASS"
           fi
 
-      - name: Test for default instrumentation resources
+      - name: Test for default instrumentation resources for Java
         run: |
-          cd integration-tests/manifests
-          kubectl apply -f ./sample-deployment.yaml
+          kubectl apply -f integration-tests/java/sample-deployment-java.yaml
           sleep 60
           kubectl get pods -A 
           kubectl describe pods -n default
-          cd ./cmd
-          go run validate_instrumentation_vars.go default default_instrumentation_env_variables.json
+          go run integration-tests/manifests/cmd/validate_instrumentation_vars.go default integration-tests/java/default_instrumentation_java_env_variables.json
 
-      - name: Test for defined instrumentation resources
+      - name: Test for defined instrumentation resources for Java
         run: |
-          cd integration-tests/manifests
-          kubectl apply -f ./sample-instrumentation.yaml
-          sleep 60
+          kubectl apply -f integration-tests/manifests/sample-instrumentation.yaml
+          sleep 5
           kubectl rollout restart deployment nginx
+          sleep 5
+          kubectl get pods -A
+          kubectl describe pods -n default
+          cd integration-tests/manifests/cmd
+          go run validate_instrumentation_vars.go default ns_instrumentation_env_variables.json     
```

Comment:
after we validate the namespaced instrumentation resource we should probably delete it via `kubectl delete instrumentation sample-instrumentation`

I'm surprised this didn't cause the python test to fail...since we would end up using the namespaced instrumentation instead of the defaults.

Reply:
I see, the default python values are incorrect which is why it passed